sumologic/resource_sumologic_partition.go

```diff
@@ -32,7 +32,7 @@ func resourceSumologicPartition() *schema.Resource {
 			"analytics_tier": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"enhanced", "basic", "cold"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"continuous", "frequent", "infrequent", "enhanced", "basic", "cold"}, false),
 				Default:      "enhanced",
```

Comment:
Update the default value to be `continuous` to match our API docs.

Reply:
The API changes it to `enhanced`. I'm thinking maybe we should delay this PR until the API is fixed.

Reply:
Or we can remove the default for now and fix that part later.